syntax = "proto3"; //声明proto的版本

package zrpc; //作用域
option cc_generic_services = true;

message ResultCode
{
    int32 result = 1;
    bytes errmsg = 2;
}

//登录请求
message LoginReq
{
    bytes name = 1;
    bytes pwd = 2;
}
message LoginAck
{
    ResultCode result = 1;
    bool success = 2;
}

service UserServiceRpc
{
    rpc Login(LoginReq) returns(LoginAck);
}
